vktrace/src/vktrace_common/optimization_function.cpp

```diff
@@ -0,0 +1,55 @@
+#include <stdio.h>
+#include <string.h>
+#include <ppl.h>
+#include <stdlib.h>

```

Comment:
Will update to make this be included only on Windows. We will work on cross-platform solution.